tests/acceptance/test_mender-connect.py

```diff
@@ -222,10 +235,10 @@ def test_mender_connect_auth_changes(
             )
 
             # 2. Change url
+            dbus_set_token_and_url(connection, "token2", "http://localhost:6000")
+            time.sleep(30)
             signal_time = qemu_system_time(connection)
-            dbus_set_token_and_url_and_emit_signal(
-                connection, "token2", "http://localhost:6000"
-            )
+            dbus_emit_signal(connection)
```

Comment:
This should stay as it was, why we need to sleep between setting state in the mock and emitting the signal?